presentation/src/main/java/com/yapp/bol/presentation/view/group/dialog/guest/GuestListAdapter.kt

```diff
@@ -0,0 +1,87 @@
+package com.yapp.bol.presentation.view.group.dialog.guest
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.yapp.bol.designsystem.R
+import com.yapp.bol.presentation.databinding.RvMemberItemBinding
+import com.yapp.bol.presentation.model.MemberInfo
+import com.yapp.bol.presentation.utils.Constant.EMPTY_STRING
+
+class GuestListAdapter(
+    private val isEnableButton: (Boolean, String) -> Unit
+) : ListAdapter<MemberInfo, GuestListAdapter.GuestViewHolder>(diff) {
+
+    var selectGuest: Int? = null
```

Comment:
p4
```suggestion
    var selectedGuest: Int? = null
```
프로퍼티 명을 명사/형용사 로 변경해보았습니다 .

Reply:
굿입니다 !

Reply:
수정 완료 ~ 5a76fc42c7903583536733b85eb1276e94b7e567